examples/basic/version.tf

```diff
@@ -6,7 +6,7 @@ terraform {
   required_providers {
     ibm = {
       source  = "IBM-Cloud/ibm"
-      version = "1.54.0"
+      version = ">= 1.65.0, < 2.0.0"
```

Comment:
we want at least 1 example using the lowest supported version, so in this case, it would now be `1.65.0` - can you update to that please?